runtime/autoload/man.vim

```diff
@@ -112,7 +106,7 @@ endfunction
 
 function! s:get_path(sect, name) abort
   if empty(a:sect)
-    let path = system(s:man_cmd.' '.s:man_find_arg.' '.shellescape(a:name))
+    let path = system([s:man_cmd, s:man_find_arg, a:name])
```

Comment:
Now that we're not using `man 2>/dev/null` will man.vim have a problem? cc @nhooyr